openshift/template.yaml

```diff
@@ -385,7 +393,11 @@ objects:
               - name: route53-creds
                 mountPath: "/etc/.aws"
                 readOnly: true
+              - name: cache-volume
```

Comment:
FWIW this isn't just the cache. The installer temp dirs will get created here as well but they will not stick around.

The term we use in the source is "working directory", maybe something similar would make this more clear.